tests/RobotFramework/tests/tedge/tedge_config_add_remove.robot

```diff
@@ -0,0 +1,59 @@
+*** Settings ***
+Resource            ../../resources/common.resource
+Library             ThinEdgeIO
+
+Suite Setup         Setup
+Suite Teardown      Get Logs
+
+Test Tags           theme:cli    theme:configuration
+
+
+*** Test Cases ***
+tedge config add/remove value in empty array type config
+    Execute Command    sudo tedge config add c8y.smartrest.templates 1
+    ${add}    Execute Command    tedge config list
+    Should Contain    ${add}    c8y.smartrest.templates=["1"]
+
+    Execute Command    sudo tedge config remove c8y.smartrest.templates 1
+    ${remove}    Execute Command    tedge config list
+    Should Contain    ${remove}    c8y.smartrest.templates=[]
+
+tedge config add/remove value in non-empty array type config
+    Execute Command    sudo tedge config set c8y.smartrest.templates 1
+    ${set}    Execute Command    tedge config list
+    Should Contain    ${set}    c8y.smartrest.templates=["1"]
+
+    Execute Command    sudo tedge config add c8y.smartrest.templates 2,3
+    ${add}    Execute Command    tedge config list
+    Should Contain    ${add}    c8y.smartrest.templates=["1", "2", "3"]
+
+    Execute Command    sudo tedge config remove c8y.smartrest.templates 1,2,3
```

Comment:
Before removing all the values, I test the case where we just remove one value, say "2".